runtime/near-vm-runner/src/imports.rs

```diff
@@ -102,7 +102,7 @@ imports! {
     sha256<[value_len: u64, value_ptr: u64, register_id: u64] -> []>,
     keccak256<[value_len: u64, value_ptr: u64, register_id: u64] -> []>,
     keccak512<[value_len: u64, value_ptr: u64, register_id: u64] -> []>,
-    #["protocol_feature_ed25519_verify", Ed25519Verify] ed25519_verify<[
+    ed25519_verify<[
```

Comment:
Here we cannot remove the flag. We must only allow the `ed25519_verify` import on the newest version. When replaying older block and someone tried to use `ed25519_verify`, we must retain the current behavior, which is to fail with an invalid import error.

Luckily, the `imports!` macro here already has a nice way of dealing with that:
```suggestion
    #[Ed25519Verify] ed25519_verify<[
```